lib/Rex/CLI.pm

```diff
@@ -451,8 +451,10 @@ CHECK_OVERWRITE: {
     my %real_groups;
 
     for my $group ( keys %groups ) {
-      my @servers = map { $_->get_servers }
+      my @servers = 
+        map { $_->get_servers }
         Rex::Group->get_group_object($group)->get_servers;
+      @servers = map { $_->name } @servers if $opts{'y'} >= 2;
```

Comment:
Perhaps it can be called `-Tc` instead and described as `List all objects for tab-completion`.